.github/workflows/reusable-module-prep-release.yml

```diff
@@ -107,8 +107,11 @@ jobs:
       - name: Update version in package.json
         if: ${{ inputs.json-file != '' }}
         id: new
+        env:
+          JSON_FILE: ${{ inputs.json-file }}
+          RELEASE_LEVEL: ${{ inputs.level }}
         run: |
-          RAW_VERSION=$(npm version "${{ github.event.inputs.level }}" --no-git-tag-version --prefix "$(dirname "${{ inputs.json-file }}")")
+          RAW_VERSION=$(npm version "${RELEASE_LEVEL}" --no-git-tag-version --tag-version-prefix="" --prefix "$(dirname "${JSON_FILE}")")
```

Comment:
This is a reasonable fix.

The `--tag-version-prefix` is an npm setting. Is there any reason why we shouldn't add this to all of our modules by default in the `.npmrc` file?

Reply:
I don't see why not. Odd that it hasn't been an issue until this recent workflow.